src/metadataExt.ts

```diff
@@ -237,18 +297,20 @@ export class MetadataModuleExt {
     private unwrapObj(obj: any): any {
         return obj[Object.keys(obj)[0]];
     }
-}
 
-type UriTranslator = (oldUri: string) => string;
+    private shouldCopyVisObj(options: ICopyDashboardOptions): boolean {
+        if (options.copyVisObj || typeof options.copyVisObj === "undefined") {
```

Comment:
Style suggestion: you can always replace conditions like

```ts
if (condition) {
  return true;
} else {
  return false;
}
```

with `return condition;` or if the types do not match with `return !!(condition);` The `!!` converts the `condition` to `boolean`.